gammapy/estimators/flux_map.py

```diff
@@ -291,13 +300,15 @@ def energy_max(self):
     def niter(self):
```

Comment:
Very good point.
I have made the unification between 'niter' and 'ncalls': 'ncalls' is then the only parameter whatever the method of parameter estimation

Reply:
Note that `niter` and `ncalls` is not necessarily the same: some optimizers require multiple function evaluations per iteration. In practice it might be hard to distinguish. However I'm +1 to unify the name. Would you mind @bkhelifi and do this is a separate PR (as @QRemy suggested as well) first?